packages/functional-tests/pages/baseTokenCode.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { BaseLayout } from './layout';
+import { expect } from '../lib/fixtures/standard';
+
+export class BaseTokenCodePage extends BaseLayout {
+  // This convention seems pretty useful. Maybe we should put it on BaseLayout.
+  checkForPath(path?: string) {
+    if (path) {
+      expect(this.page.url()).toContain(path);
+    }
```

Comment:
We might want to return an error if the path is not as expected, or we'll miss instances of page methods being used on the wrong page.

Reply:
Good call. An general error with an appropriate message is now thrown.